pom.xml

```diff
@@ -319,12 +319,12 @@ under the License.
       <dependency>
         <groupId>org.apache.maven.shared</groupId>
         <artifactId>maven-shared-utils</artifactId>
-        <version>3.2.1</version>
+        <version>3.3.4-SNAPSHOT</version>
```

Comment:
next release of maven-shared-utils should probably be 3.4.0 since you've added new public API and features.

Reply:
I'll upgrade the PR with the released version once https://github.com/apache/maven-shared-utils/pull/69 is merged and releases, whatever the exact version is.

Reply:
@gnodet Took some time, but we managed to release maven-shared-utils 3.3.4 yesterday. I believe [your PRs](https://github.com/apache/maven/pulls/gnodet) are now unblocked.